pkg/catalog/available_updates.go

```diff
@@ -31,10 +31,10 @@ func GetAvailableUpdates(catalogClient CatalogClient, catalogName, profileName,
 		}
 		return nil, fmt.Errorf("failed to fetch available updates for profile, status code %d", statusCode)
 	}
-	var profiles []profilesv1.ProfileCatalogEntry
+	var profiles grpcProfileCatalogEntryList
```

Comment:
need to update to reference profiles?

Reply:
I haven't applied the release yet. :))) Hold your horses. :D

Reply:
Now, it's applied. :D